sdk/keyvault/azkeys/client.go

```diff
@@ -123,12 +133,13 @@ type CreateKeyResponse struct {
 
 // creates CreateKeyResponse from generated.KeyVaultClient.CreateKeyResponse
 func createKeyResponseFromGenerated(g generated.KeyVaultClientCreateKeyResponse) CreateKeyResponse {
+	vaultURL, name, version := shared.ParseID(g.Key.Kid)
 	return CreateKeyResponse{
 		Key: Key{
-			Properties: keyPropertiesFromGenerated(g.Attributes),
+			Properties: keyPropertiesFromGenerated(g.Attributes, g.Key.Kid, name, version, g.Managed, vaultURL, g.Tags),
```

Comment:
Someday you'll probably want to pass in a struct or something with named fields, to avoid accidental tranposition with params.

Reply:
Agreed. I could move some of the parsing into the method at a later date